mifosng-android/src/main/java/com/mifos/objects/collectionsheet/ClientCollectionSheet.java

```diff
@@ -14,6 +15,7 @@
     private int clientId;
     private String clientName;
 
+    @Nullable
     private ArrayList<LoanCollectionSheet> loans;
```

Comment:
`LoanCollectionSheet` refers to the Loan related to the CollectionSheet. Again, like ClientCollectionSheet, I named this so to discern it from the normal Loans. It's called `loans` as this is the identifier which is received in the JSON response.